services/iam/drivers/IamAccount.py

```diff
@@ -298,3 +304,24 @@ def _checkConfigEnabled(self):
             self.results['PartialEnableConfigService'] = [-1, ', '.join(badResults)]
         else:
             return
+
+    def _checkSCPEnabled(self):
+        # Run this check only when AWS Organization is activated in the account
+        if self.organizationIsEnabled == True:
+            try:
+                # Get organization root ID
+                roots = self.orgClient.list_roots()
+                root_id = roots['Roots'][0]['Id']
+                
+                policies = self.orgClient.list_policies_for_target(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The API method `list_policies_for_target` returns paginated results instead of all results. Consider using the pagination API or checking one of the following keys in the response to verify that all results were returned: `IsTruncated`, `NextToken`.

[Learn more](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html)